ee/ucp/deploy-application-package.md

```diff
@@ -35,19 +35,19 @@ To create a stack in the UCP web interface, follow these steps:
 
 6. Select either **Compose File** or **App Package** for the **Application File Mode**.
 7. Select **Next**.
-8. If you selected Compose file, enter or upload your `docker-compose.yml` in **2. Add Application File**.
+8. If you selected Compose file, enter or upload your `docker-compose.yml` in the **Create Application** window.
 
     ![Provide docker-compose.yml in UCP](/ee/ucp/images/ucp-stack-compose.png)
 
-   or if you selected **App Package**, enter or upload the application package in the single-file format.
+   If you selected **App Package**, enter or upload the application package in the single-file format.
 
     ![Provide application package in UCP](/ee/ucp/images/ucp-stack-package.png)
 
 9. Select **Create**.
 
 ## Single-file format example
 
-Here is an example of a single-file application package:
+The following file is an example of a single-file application package:
```

Comment:
Capitalize 'single'.

Reply:
Can you explain this comment in more detail?